### PR TITLE
GH: Extend the issue template to request verbose debug logs for non-crash issues.

### PR DESCRIPTION
**Pull request by NiLuJe**
Because it's literally 90% of my first replies :/.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/9585)

### CODE REVIEW

**Comment by poire-z** [comment]
`To do so, go to [Tools] → More tools → Developer options, `
May be mention that this menu is available only in File browser - as it's 80% of our second replies :)

**Comment by mergen3107** [comment]
While we are here, shouldn’t we also ask to replicate the same issue after enabling verbose logging? IF I understood it correctly, verbose logs do not verbose log events happened before they were enabled. Is this right? :D

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -15,20 +15,20 @@ assignees: ''
...
 * `applications/koreader/` directory for Pocketbook
 
-
-Android logs are kept in memory. Please go to Help → Bug Report to save these logs to a file.
+Android logs are kept in memory. Please go to [Menu] → Help → Bug Report to save these logs to a file.
 
 Please try to include the relevant sections in your issue description.
-You can upload the whole `crash.log` file on GitHub by dragging and
-dropping it onto this textbox.
+You can upload the whole `crash.log` file on GitHub by dragging and dropping it onto this textbox.
```
Comment (Frenzie) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Also zipped if necessary? Not sure what the filesize limit is otoh (10 MB?) but zipping does cut it very drastically.

**Comment by hius07** [comment]
Not to forget to correct this after we get rid of the "More tools" menu item (long awaiting step).

**Comment by poire-z** [comment]
> after we get rid of the "More tools" menu item (long awaiting step).

I may have missed this discussion :) What's the idea / alternative ?

**Comment by hius07** [comment]
> I may have missed this discussion

https://github.com/koreader/koreader/pull/6101#discussion_r417475214

**Comment by poire-z** [comment]
OK, that was 2 years ago, my crappy memory can be pardonned :)

Until we go with submenus per topic (auto*, device management, reading tools, developper tools, connectivity stuff) and the ability for a user to move their preferred tools up a menu level (so they are in the first menu) , isn't it just a matter of sorting them (main | more tools) or even now into submenus ? But if you disable many, you'll get submenus with a single item.... which sucks. So, not easy. May be create a dedicated issue so this can be discussed?

**Pull request merged**